docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -59,6 +59,18 @@ connections.
 `max_cost`:::
 Total size of the cache.
 
+`server.timeouts`::
+`checkin_timestamp`:::
+How often Fleet Server updates the "last activity" field for each agent.
+Defaults to `30s`. When deploying higher scales of agents, increasing this
+setting may improve performance. If set higher than `2m`, most agents will be
```

Comment:
In 8.4.0 we're changing this UI timeout to 10m instead of 2m. We may want to make the UI configurable too though